CMakeLists.txt

```diff
@@ -98,6 +99,7 @@ endif()
 add_compile_options("$<$<C_COMPILER_ID:MSVC>:/utf-8>")
 add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
 add_library(ortcustomops_static STATIC ${TARGET_SRC})
+target_include_directories(ortcustomops_static PUBLIC tokenizer)
```

Comment:
so the tokenizer cannot be disabled in the compiling now?

Reply:
Emm... We can discuss this more in the meeting. You can refer this question:[How to include all objects of an archive in a shared object?](https://stackoverflow.com/questions/2763988/how-to-include-all-objects-of-an-archive-in-a-shared-object?rq=1)